docs/source/locale/ja/LC_MESSAGES/index.po

```diff
@@ -0,0 +1,93 @@
+# This file is distributed under the same license as the Podman package.
+# Translators:
+# Takuya Nishimura <tanishim@redhat.com>, 2023.
+msgid ""
+msgstr ""
+"Project-Id-Version: Podman \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-08-29 22:22+0900\n"
+"PO-Revision-Date: 2023-08-29 22:22+0900\n"
+"Last-Translator: Takuya Nishimura <tanishim@redhat.com>\n"
+"Language: ja\n"
+"Language-Team: ja\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.12.1\n"
+
+#: ../../source/index.rst:15
+msgid "Contents:"
+msgstr "コンテンツ"
+
+#: ../../source/index.rst:4
+msgid "What is Podman?"
+msgstr "Podmanとは？"
+
+#: ../../source/index.rst:5
+msgid ""
+"Podman_ is a daemonless, open source, Linux native tool designed to make "
+"it easy to find, run, build, share and deploy applications using Open "
+"Containers Initiative (OCI_) Containers_ and `Container Images`_. Podman "
+"provides a command line interface (CLI) familiar to anyone who has used "
+"the Docker `Container Engine`_. Most users can simply alias Docker to "
+"Podman (`alias docker=podman`) without any problems. Similar to other "
+"common `Container Engines`_ (Docker, CRI-O, containerd), Podman relies on"
+" an OCI compliant `Container Runtime`_ (runc, crun, runv, etc) to "
+"interface with the operating system and create the running containers. "
+"This makes the running containers created by Podman nearly "
+"indistinguishable from those created by any other common container "
+"engine."
+msgstr ""
+"Podman_ は、Open Containers Initiative (OCI_) コンテナ_ や `コンテナイメージ`_ を使用した"
+"アプリケーションの検索、実行、ビルド、共有、デプロイが容易にできるように設計された"
+"デーモンレスのオープンソースLinuxネイティブツールです。Podmanは、Docker `コンテナエンジン`_ "
+"の利用者ほとんどに馴染みのあるコマンドラインインターフェイス (CLI)を提供します。ほと"
+"んどのユーザは、DockerをPodmanにエイリアスする(`alias docker=podman`)だけで何の"
+"問題もなく利用できます。他の一般的な `コンテナエンジン`_ (Docker、CRI-O、containerd)"
+"と同様に、PodmanはOCI準拠の `コンテナランタイム`_ (runc、crun、rinvなど)に基づき、"
```

Comment:
Looking at the English text, `rinv` should be `runv`.
Also, it is not about translation, I think `runv` is not necessary because it is not a major container runtime today.

Reply:
thank you. fixed the typo